### DISCUSSION TITLE
for telegram cloud storage

### DISCUSSION THREAD

**Discussion by safa7** [revenkroz/telegram-web-app-bot-example#24] [Q&A] [answered]
in the bot I can see example for TCS but I can't find the code in html page, can you give me an example TCS usage pls.
![tcs](https://github.com/revenkroz/telegram-web-app-bot-example/assets/10710749/3f257835-bb36-435b-862b-f44f62ca4ee2)

**Comment by revenkroz** [comment]
Ok, I'm going to update repo a bit later

**Comment by safa7** [comment]
plz dont forget updates

**Reply by pd5200** [reply]
hello i already wait for 2 month

**Comment by revenkroz** [comment, marked as answer]
Done
